spring-cloud-gcp-autoconfigure/src/main/java/org/springframework/cloud/gcp/autoconfigure/secretmanager/GcpSecretManagerProperties.java

```diff
@@ -39,17 +36,6 @@
 	 */
 	private String projectId;
 
-	/**
-	 * Defines a prefix String that will be prepended to the environment property names
-	 * of secrets in Secret Manager.
-	 */
-	private String secretNamePrefix = "";
```

Comment:
Would it be so bad to let users overwrite the prefix?

Reply:
I think it can only hurt users if they choose a bad prefix. It needs to be a unique/special string that lets us exit early so we know that it's definitely not a property we need to parse and attempt to query.